### PR TITLE
v10 Release

### PR DESCRIPTION
**Pull request by joshblack**
# DO NOT MERGE

Items left to do:

- [x] https://github.com/IBM/carbon-components/pull/2237
- [x] ~Fix focus-outline issue for data-table-v2 scss, fix in #2213~

### CODE REVIEW

**Comment by joshblack** [comment]
[`v10.0.0-beta.1`](https://github.com/IBM/carbon-components/releases/tag/v10.0.0-beta.1) published

File: package.json [hunk 1]
```diff
@@ -172,10 +172,12 @@
...
     "scss/**/*",
     "src/**/*",
-    "umd/**/*"
+    "umd/**/*",
+    "!**/*.config.js",
+    "!**/__tests__/**/*"
   ],
   "scripts": {
-    "build": "gulp build",
+    "build": "gulp build && gulp html:source -e",
```
Comment (asudoh) [inline package.json]: Shouldn't be necessary as long as we flip feature flags, but definitely let me know if you find otherwise!

**Review by asudoh — COMMENTED** [review]
Thanks @joshblack for creating this, one question - Are you planning to create PRs for documentation/test updates?

**Comment by joshblack** [comment]
@asudoh definitely can! I don't mind at all 👍 Some of the updates will have to happen in this PR, just because we're swapping flags.

File: package.json (hunk 1 above)
Comment (joshblack) [reply package.json]: @asudoh it seems like the default build doesn't generate this, unfortunately. Seems like something is going on 🤔

**Comment by joshblack** [comment]
Waiting on https://github.com/IBM/carbon-components/pull/2213 for last failing test

File: package.json (hunk 1 above)
Comment (asudoh) [reply package.json]: OK good catch @joshblack! Created: #2215

**Comment by asudoh** [comment]
@joshblack Cool, also created #2214, which seems to pass `ci-check` with feature flags flipped - Wanted to see if it's good, or you still want to create a Sass test render helper for classic (seen this PR), etc. Thanks!

**Comment by joshblack** [comment]
[`v10.0.0-rc.0`](https://github.com/IBM/carbon-components/releases/tag/v10.0.0-rc.0) published

**Comment by vpicone** [comment]
Opened a PR here to rename data table: https://github.com/joshblack/carbon-components/pull/1

I think it might be good for a couple of people to sanity check @joshblack @asudoh

**Review by alisonjoseph — APPROVED** [review]

**Review by jeanservaas — APPROVED** [review]
LGTM!

**Review by jnm2377 — APPROVED** [review]

File: src/globals/js/components.js [hunk 2]
```diff
@@ -16,11 +15,8 @@ export { default as Loading } from '../../components/loading/loading';
 export { default as InlineLoading } from '../../components/inline-loading/inline-loading';
 export { default as Dropdown } from '../../components/dropdown/dropdown';
 export { default as NumberInput } from '../../components/number-input/number-input';
-export { default as DataTable } from '../../components/data-table/data-table';
 export { default as DataTableV2 } from '../../components/data-table-v2/data-table-v2';
```
Comment (asudoh) [inline src/globals/js/components.js:18]: Not sure where we are going, but wanted to see if team wants this is in sync with the latest.

**Pull request merged**